packages/lab/stories/progress/progress.linear.stories.tsx

```diff
@@ -21,142 +21,38 @@ const ProgressWithControls = ({
     useProgressingValue();
   return (
     <div className="salt-ProgressWithControls-root">
-      <div className="salt-ProgressWithControls-controls">
+      <FlowLayout className="salt-ProgressWithControls-controls" gap={1}>
```

Comment:
nit: none of our class names are like this, i would porbably change them to be something similar (literally just root, controls would be fine, doesnt need prefixes since its just a story)